### PR TITLE
Add Circle CI, a README, and Badges

### PR DESCRIPTION
**Pull request by galcantara-asapp**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.1'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```
Comment (goddardcm) [inline build.gradle:21]: This change seems unrelated to CircleCI.

**Review by matheus-asapp — APPROVED** [review]
